tests/components/rfxtrx/test_sensor.py

```diff
@@ -34,6 +38,27 @@ async def test_one_sensor(hass, rfxtrx):
     assert state.attributes.get("unit_of_measurement") == TEMP_CELSIUS
 
 
+@pytest.mark.parametrize(
+    "state,event",
+    [["18.4", "0a520801070100b81b0279"], ["17.9", "0a52085e070100b31b0279"]],
+)
+async def test_state_restore(hass, rfxtrx, state, event):
+    """State restoration."""
+
+    entity_id = "sensor.wt260_wt260h_wt440h_wt450_wt450h_07_01_temperature"
+
+    mock_restore_cache(hass, [State(entity_id, state, attributes={ATTR_EVENT: event})])
+
+    assert await async_setup_component(
+        hass,
+        "rfxtrx",
+        {"rfxtrx": {"device": "abcd", "devices": {"0a520801070100b81b0279": {}}}},
+    )
+    await hass.async_block_till_done()
+
+    assert hass.states.get(entity_id).state == state
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5872960)